Jenkinsfile.aws-test

```diff
@@ -59,6 +59,9 @@ node(NODE) {
                     fi
 
                     cp ${dirpath}/aws-${AWS_REGION}-smoketested.json ${images}/aws-${AWS_REGION}-tested.json
+                    aws ec2 create-tags \
+                        --resources \${ami_intermediate} \
```

Comment:
Minor: let's drop the escape `\` here? (I think it'd work either way since it's both a Groovy var and an env var, but just to be consistent with the next command).